### DISCUSSION TITLE
Network Card and Audio Gridder

### DISCUSSION THREAD

**Discussion by AMWMusic** [apohl79/audiogridder#1280] [Q&A]
I would like to ask, if the performance of Audio Gridder can be ameliorated, when using network cards that can work faster than 1Gbit/s.

**Comment by apohl79** [comment]
I would say it depends. Performance depends a many factors, but a good network connection is essential.

**Reply by AMWMusic** [reply]
Thank you for your reply! Measurement has shown that only one of my onboard network cards can deliver about 0,8 Gbit/s, the other two deliver about 0.3 - 0.4 Gbit/s. I now have ordered two network cards, that are much faster and I hope that the performance of Audio Gridder can be ameliorated with that. Regarding MIDI data I don't have any issues. When it comes to audio, data transfer may increase, as it seems.
